### PR TITLE
Documentation and framework linking clean up

### PR DESCRIPTION
**Pull request by ejensen**
This change improves the HeaderDoc compatibility of the `FBShimmerDirection` enum. 
It also removes linking to frameworks such as CoreData and AppKit.

### CODE REVIEW

File: Examples/Examples.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -21,6 +21,7 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
+		077ACA6D1CBD6B7200B3A708 /* UIKit.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = UIKit.framework; path = Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS9.3.sdk/System/Library/Frameworks/UIKit.framework; sourceTree = DEVELOPER_DIR; };
```
Comment (grp) [inline Examples/Examples.xcodeproj/project.pbxproj]: The path here looks wrong: shouldn't it be SDK relative in order to avoid hardcoding an SDK version?

**Comment by grp** [comment]
Looks good besides the one comment — will merge if you update that. Thanks!

File: Examples/Examples.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ejensen) [inline Examples/Examples.xcodeproj/project.pbxproj]: Commit 0711a87 corrects the path. 
This was caused by the SDK defaulting to the Mac OS SDK. Commit 5fc7ee0 sets the SDK to iOS

**Comment by grp** [comment]
Thanks!

**Pull request merged**